CHANGELOG.md

```diff
@@ -3,6 +3,19 @@
 Documentation for rocRAND is available at
 [https://rocm.docs.amd.com/projects/rocRAND/en/latest/](https://rocm.docs.amd.com/projects/rocRAND/en/latest/)
 
+
+## (Unreleased) rocRAND 3.2.0 for ROCm 6.4
+
+### Added
+* Added regression tests to `rtest.py`. Regression tests are a subset of tests that caused hardware problems for past emulation environments.
+  * Can be ran with `python rtest.py [--emulation|-e|--test|-t]=regression`
+* Added smoke test options, which runs a subset of the unit tests and ensuring that less than 2gb of VRAM will be used
+  * Smoke tests can be ran with `[--emulation|-e|--test|-t]=smoke`
+* Added `--emulation` option added for `rtest.py`
+
+### Changed
+* `--test|-t` no longer a required flag for `rtest.py`, instead user can use either `--emulation|-e` or `--test|-t` but not both
```

Comment:
I added some edits to this section in #573. I think you'll want to merge those in here.